.github/workflows/ci.yml

```
@@ -6,7 +6,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Might as well hop on Jammie immediately 😄 

```suggestion
    runs-on: ubuntu-22.04
```